python/pipeline/experiment.py

```diff
@@ -359,28 +359,17 @@ class SurgeryOutcome(dj.Lookup):
     ]
 
 
-@schema
-class SurgeryQuality(dj.Lookup):
-    definition = """ # Surgery qualities used in experiment.Surgery
-    surgery_quality              : varchar(32)         # Possible qualities of surgeries performed on mice
-    """
-    contents = [
-        ['Good'],
-        ['Okay'],
-        ['Bad']
-    ]
-
-
 @schema
 class Surgery(dj.Manual):
     definition = """ # List of surgeries performed on mice
     -> mice.Mice
     date                         : date                   # Date surgery was performed
```

Comment:
Maybe instead of having the timestamp of the surgery as primary key, you can use an int (surgery_id) that just increases for every surgery. Not sure whether it's recommended to have times in primary keys (quick Google search is ambivalent). It can still be a secondary attribute.
It also makes restrictions nicer `Surgery & {'animal_id': 123, 'surgery_id': 0}` rather than `Surgery & {'animal_id': 123, 'surgery_id': '2019-08-03'}` which requires knowing date/time/timestamp formatting (for instance, I'm not sure the restriction I just wrote will work or whether 08 is the month or the date)

Reply:
Also, not sure about your use of date and time as diff attributes, I would just use a timestamp as we do pretty much everywhere we want to record time.

Reply:
Restrictions would be difficult to perform without knowing the exact surgery_id ahead of time, so I'd end up using the secondary 'date' key for the restrictions in most use cases. I can add a description to the date key saying "YYYY-MM-DD format" to help clear up that confusion.